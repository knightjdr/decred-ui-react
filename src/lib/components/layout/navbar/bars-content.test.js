import BarsContent from './bars-content';

describe('BarsContent', () => {
  test('Renders', () => {
    const content = BarsContent();
    expect(content).toMatchSnapshot();
  });
});
