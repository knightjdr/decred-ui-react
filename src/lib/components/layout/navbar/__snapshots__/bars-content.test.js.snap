// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarsContent Renders 1`] = `
<svg
  height="30"
  viewBox="0 0 448 512"
  width="26.25"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    className="decred-ui-navbar-menu-bars"
    d="M436 124H12c-6.627 0-12-5.373-12-12V80c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12zm0 160H12c-6.627 0-12-5.373-12-12v-32c0-6.627 5.373-12 12-12h424c6.627 0 12 5.373 12 12v32c0 6.627-5.373 12-12 12z"
  />
</svg>
`;
